package.json

```diff
@@ -89,29 +103,56 @@
     "gatsby-plugin-catch-links": "^2.1.26",
     "gatsby-plugin-google-analytics": "^2.1.36",
     "gatsby-plugin-manifest": "^2.2.42",
+    "gatsby-plugin-postcss": "^2.2.0",
     "gatsby-plugin-react-helmet": "^3.1.22",
     "gatsby-plugin-sentry": "^1.0.1",
+    "gatsby-plugin-sharp": "^2.5.2",
     "gatsby-plugin-sitemap": "^2.2.27",
     "gatsby-plugin-styled-components": "^3.1.19",
+    "gatsby-plugin-svgr": "^2.0.2",
+    "gatsby-plugin-twitter": "^2.2.0",
+    "gatsby-plugin-typescript": "^2.2.5",
+    "gatsby-plugin-webpack-bundle-analyzer": "^1.0.5",
     "gatsby-remark-autolink-headers": "^2.1.24",
     "gatsby-remark-copy-linked-files": "^2.1.37",
+    "gatsby-remark-embed-gist": "^1.1.9",
+    "gatsby-remark-embedder": "^1.14.0",
     "gatsby-remark-external-links": "^0.0.4",
+    "gatsby-remark-images": "^3.2.0",
     "gatsby-remark-prismjs": "^3.3.31",
+    "gatsby-remark-relative-images": "^0.3.0",
+    "gatsby-remark-responsive-iframe": "^2.3.0",
     "gatsby-remark-smartypants": "^2.1.21",
     "gatsby-source-filesystem": "^2.1.48",
-    "gatsby-transformer-remark": "^2.6.53",
+    "gatsby-transformer-remark": "^2.6.59",
+    "gatsby-transformer-sharp": "^2.4.0",
+    "hast-util-select": "^4.0.0",
     "husky": "^4.0.10",
     "jest": "^24.9.0",
     "lint-staged": "^10.0.0",
+    "postcss-custom-media": "^7.0.8",
+    "postcss-custom-properties": "^9.1.1",
+    "postcss-mixins": "^6.2.3",
+    "postcss-nested": "^4.2.1",
     "prettier": "^1.19.1",
-    "pretty-quick": "^2.0.1"
+    "pretty-quick": "^2.0.1",
+    "rehype-parse": "^6.0.2",
+    "rehype-stringify": "^6.0.1",
+    "remark": "^11.0.2",
+    "remark-html": "^10.0.0",
+    "stylelint": "^13.2.1",
+    "stylelint-config-standard": "^20.0.0",
+    "typescript": "^3.8.3"
+  },
+  "resolutions": {
+    "@types/react-native": "link:./.tmp-empty-package"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn format-staged && yarn lint-staged"
+      "pre-commit": "yarn format-staged && tsc --noEmit && yarn lint-staged"
     }
   },
   "lint-staged": {
-    "*.{js,json}": "eslint"
+    "*.{js,json,ts,tsx}": "eslint --quiet"
```

Comment:
is it temporary?

Reply:
I mean `--quite`?

Reply:
It was temporary too, yes. Refactored remaining files with warnings and removed this option